llvm/lib/Analysis/CtxProfAnalysis.cpp

```diff
@@ -202,6 +202,13 @@ InstrProfCallsite *CtxProfAnalysis::getCallsiteInstrumentation(CallBase &CB) {
   return nullptr;
 }
 
+InstrProfIncrementInst *CtxProfAnalysis::getBBInstrumentation(BasicBlock &BB) {
+  for (auto &I : BB)
```

Comment:
I think explicitly typing `Instruction` would improve readability a bit.

Reply:
why, this is a typical iteration pattern for BBs (doing a very naive search though llvm/lib for either patterns - i.e. `for (auto &I : BB)` or `for (Instruction &I : BB)` reveals about the same nr of instances.